src/components/Header/Header.tsx

```diff
@@ -4,7 +4,7 @@ import './Header.scss';
 export const Header = () => {
   return (
     <header className="header" id="header">
-      <a className="logo" href="/"></a>
+      <a className="logo" href="/">{}</a>
```

Comment:
Rather than applying styles to logo class and add the image trough CSS, you should add an img tag and style it's class.

If you add the image trough CSS - background-image, - pseudo-elements (::after & ::before), it won't be discoverable for SEO and thus that image won't be displayed in search results.

And this should be done for the entire project and for every image or illustrations, in this case the logo (except specific cases, like an icon system)

```suggestion
      <a className="logo" href="/">
        <img
          className="logo__image"
          src="../../assets/icons/header/Logo--Name.svg"
          alt="logo name"
        />
      </a>
```